### DISCUSSION TITLE
A way to silence module not found

### DISCUSSION THREAD

**Discussion by SyntheticGoop** [yarnpkg/berry#2540] [Q&A] [answered]
Is there a way to silence MODULE_NOT_FOUND errors? I know the objective of yarn is to embrace pnp, and therefore some persuasion is necessary to make users switch. However, in some cases you will have to fall back to pnp loose mode which results in a wall of errors.

While you can ignore this most of the time, there are some cases where it is just polluting the command line.

My current issue stems from using nuxt (which was made partially compatible with pnp in 2.15.0). When using the command line dev server, an entire wall of errors covers up the main warnings from nuxt.

I did a little digging and it seems the MODULE_NOT_FOUND errors don't correspond to any silenceable error code for logFilters. Neither does doing a text match for each error seem to work.

What I am trying to do is run `pnpModel: loose` and `pnpFallbackMode: all` (which is effectively node_modules without node_modules) without the pesky warnings since its still a space saving to run it like that.

**Comment by merceyz** [comment, marked as answer]
You can set `PNP_DEBUG_LEVEL=0` to silence them, it was added in https://github.com/yarnpkg/berry/pull/2343 so you'll need to get Yarn 3 from sources using
```
yarn set version from sources
```

**Reply by SyntheticGoop** [reply]
Awesome! Using it right now. Hope yarn 3 lands some time soon so we don't have to install from sources.
